apps/bbsync/query.py

```diff
@@ -289,3 +293,20 @@ def generate_srt_notes(self):
         """
         srt_notes_builder = SRTNotesBuilder(self.flaw, self.old_flaw)
         self._query["cf_srtnotes"] = srt_notes_builder.content
+
+    def generate_comment(self):
+        with transaction.atomic():
```

Comment:
All the API calls are atomic by default and the Django is responsible for it
https://github.com/RedHatProductSecurity/osidb/blob/master/config/settings_prod.py#L74